src/components/Modal/User.vue

```diff
@@ -1,8 +1,9 @@
 <template>
   <UiModal :open="open" @close="$emit('close')">
     <div class="m-4 mb-0 text-center">
-      <Avatar :address="address" size="64" class="mb-4" />
-      <h3 v-text="_shorten(address)" />
+      <Avatar :profile="profile" :address="address" size="64" class="mb-4" />
+      <h3 v-if="profile && profile.name" class="mt-3" v-text="profile.name" />
+      <h3 v-else v-text="_shorten(address)" />
```

Comment:
Space is missing between the avatar and h3 when there is no profile
```suggestion
      <h3 v-else v-text="_shorten(address)" class="mt-3" />
```